cmd/metrics-scraper/app/scrape/scrape.go

```diff
@@ -0,0 +1,232 @@
+package scrape
+
+import(
+	"time"
+	"database/sql"
+	"log"
+	"sync"
+	"context"
+	  
+	"fmt"
+	"net/http"
+ 	"github.com/gin-gonic/gin"
+	db "github.com/karmada-io/dashboard/cmd/metrics-scraper/app/db"	 
```

Comment:
the pkg alias is useless, just use `github.com/karmada-io/dashboard/cmd/metrics-scraper/app/db` directly